### PR TITLE
Mes 6955 rehydration issues

### PR DESCRIPTION
**Pull request by andrewsetterfield**
## Description

MES-6955
Comms screen email fixes and WRTC eyesight fixes

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [ ] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

### CODE REVIEW

File: src/app/pages/communication/communication.page.ts [hunk 1]
```diff
@@ -216,10 +224,14 @@ export class CommunicationPage extends PracticeableBasePageComponent implements
   }
 
   dispatchCandidateChoseNewEmail(communicationEmail: string): void {
-    this.setCommunicationType(CommunicationPage.email, CommunicationPage.updatedEmail);
-    this.store$.dispatch(communicationPreferencesActions.CandidateChoseEmailAsCommunicationPreference(
-      communicationEmail, CommunicationPage.email,
-    ));
+    console.log('this.candidateProvidedEmail', this.candidateProvidedEmail);
+    console.log('this.communicationEmail', this.communicationEmail)
```
Comment (gittins7) [inline src/app/pages/communication/communication.page.ts]: Can you remove these console logs please

File: src/app/pages/communication/communication.page.ts [hunk 2]
```diff
@@ -313,7 +329,13 @@ export class CommunicationPage extends PracticeableBasePageComponent implements
   }
 
   getNewEmailAddressValue() {
+    console.log('getNewEmailAddressValue');
```
Comment (gittins7) [inline src/app/pages/communication/communication.page.ts]: Can you remove this console log please

File: src/app/pages/communication/communication.page.ts [hunk 3]
```diff
@@ -313,7 +329,13 @@ export class CommunicationPage extends PracticeableBasePageComponent implements
...
     return this.candidateProvidedEmail === this.communicationEmail ? '' : this.communicationEmail;
   }
 
+  setUserAction(action: userActions) {
+    this.userAction = action
+    console.log('lastClicked', action);
```
Comment (gittins7) [inline src/app/pages/communication/communication.page.ts]: Can you remove this console log please

File: src/app/pages/communication/communication.page.ts [hunk 4]
```diff
@@ -216,10 +224,14 @@ export class CommunicationPage extends PracticeableBasePageComponent implements
   }
 
   dispatchCandidateChoseNewEmail(communicationEmail: string): void {
-    this.setCommunicationType(CommunicationPage.email, CommunicationPage.updatedEmail);
-    this.store$.dispatch(communicationPreferencesActions.CandidateChoseEmailAsCommunicationPreference(
-      communicationEmail, CommunicationPage.email,
-    ));
+    console.log('this.candidateProvidedEmail', this.candidateProvidedEmail);
+    console.log('this.communicationEmail', this.communicationEmail)
+    if(this.userAction !== userActions.provided) {
```
Comment (andrewsetterfield) [inline src/app/pages/communication/communication.page.ts]: this is the bit that needs tweaking.  The obvious is this.candidateProvidedEmail !== this.communicationEmail which gives needed functionality, but doesnt update the state

**Review by gittins7 — CHANGES_REQUESTED** [review]

File: src/app/pages/communication/communication.page.ts [hunk 5]
```diff
@@ -216,10 +217,12 @@ export class CommunicationPage extends PracticeableBasePageComponent implements
   }
 
   dispatchCandidateChoseNewEmail(communicationEmail: string): void {
-    this.setCommunicationType(CommunicationPage.email, CommunicationPage.updatedEmail);
-    this.store$.dispatch(communicationPreferencesActions.CandidateChoseEmailAsCommunicationPreference(
-      communicationEmail, CommunicationPage.email,
-    ));
+    if(this.isNewEmailSelected()) {
```
Comment (tomcrawleyy) [inline src/app/pages/communication/communication.page.ts]: being pedantic here but how did this line pass linting checks?

Comment (andrewsetterfield) [reply src/app/pages/communication/communication.page.ts]: ive not idea why it didnt pick it up...will take a look

Comment (andrewsetterfield) [reply src/app/pages/communication/communication.page.ts]: resolved

**Review by tomcrawleyy — APPROVED** [review]

**Review by gittins7 — APPROVED** [review]

**Pull request merged**